src/components/Accounts/PendingAccounts.jsx

```diff
@@ -4,60 +4,149 @@ import { Table, Thead, Tbody, Tr, Th, Td, TableContainer, Button } from '@chakra
 import { Checkbox } from '@chakra-ui/react'
 import PropTypes from 'prop-types';
 
-const PendingAccounts = ( {accountType} ) => {
+const PendingAccounts = ( {accountType, setHasPendingAccounts} ) => {
     const [pendingAccounts, setPendingAccounts] = useState([]);
+    const [individualChecked, setIndividualChecked] = useState(new Array(pendingAccounts.length).fill(false));
+    const [checkedAccountIds, setCheckedAccountIds] = useState([]);
+    const [accountStatus, setAccountStatus] = useState({});
+
 
     useEffect(() => {
         const renderTable = async () => {
-            const { data } = await NPOBackend.get('/users/pending-accounts');
+            const { data } = await NPOBackend.get('/users/pending-accounts', {params: {accountType: accountType}});
             setPendingAccounts(data);
+            console.log("ELNGLKNKJ: " + data);
+            setHasPendingAccounts(data.length !== 0);
         };
         renderTable();
+    }, [])
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'accountType' and 'setHasPendingAccounts'. Either include them or remove the dependency array. If 'setHasPendingAccounts' changes too often, find the parent component that defines it and wrap that definition in useCallback.